src/widgets/mobile-view/ui/mobile-view.module.scss

```
@@ -0,0 +1,30 @@
+.mobile-view {
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  gap: 16px;
+  align-items: baseline;
+  justify-content: flex-start;
+
+  width: 100%;
+  padding-bottom: 16px;
+}
+
+.category-link {
+  font-size: 24px;
+  font-weight: $font-weight-bold;
+}
+
+.divider {
+  width: 100%;
+  border-color: $color-gray-600;
+  border-bottom: 1px solid;
```

```suggestion
  border-bottom: 1px solid $color-gray-600;
```
Why do we need 2 border properties here? Maybe we can write it as one

---

It's not entirely obvious that border-color inherits the current color by default, although it works. Maybe we should then explicitly specify what we want to inherit and from where when adding an additional class with a theme

`  border-bottom: 1px solid currentColor;`

https://www.w3.org/TR/css-backgrounds-3/#border-color
https://caniuse.com/?search=currentColor